testing/tests/DevExpress.ui.widgets/speedDialAction.tests.js

```diff
@@ -339,4 +338,54 @@ QUnit.module("check action buttons click args", (hooks) => {
     });
 });
 
+QUnit.module("check action buttons events", (hooks) => {
+    hooks.afterEach(() => {
+        $("#fab-one").dxSpeedDialAction("instance").dispose();
+        $("#fab-two").dxSpeedDialAction("instance").dispose();
+    }),
+
+    test("trigger and args", (assert) => {
+        $("#fab-one")
+            .dxSpeedDialAction()
+            .dxSpeedDialAction("instance")
+            .on("contentReady", (e) => {
+                assert.ok(e.component, "first SDA content ready component in args");
```

Comment:
Is it enough to ensure that these properties are defined?

Reply:
Added more checks